standalone/examples/cesm_t232_with_MARBL/MOM_override

```diff
@@ -0,0 +1,25 @@
+! Parameter changes to run CESM-MOM6 config in standalone mode
+
+#override INPUTDIR = /glade/work/altuntas/mom6.standalone.runs/cesm/INPUT/t232
```

Comment:
Instead of my personal work space, we should probably place all these input files under inputdata?

Reply:
good catch! This has been addressed in 7618bcd